# Builds fine without makeinfo (unless you try to build the
# documentation).

Index: configure.ac
--- configure.ac.orig
+++ configure.ac
@@ -382,7 +382,7 @@ if test "$USE_EXTRACTPDFMARK" != "yes"; then
         [" can significantly reduce the size of the final PDF files.)"])
 fi
 
-STEPMAKE_PROGS(MAKEINFO, makeinfo, REQUIRED, 6.1)
+STEPMAKE_PROGS(MAKEINFO, makeinfo, OPTIONAL, 6.1)
 TEXI2HTML_REQUIRED=$DOCUMENTATION_REQUIRED
 if test "$ENABLE_TEXI2HTML" = "no"; then
   TEXI2HTML_REQUIRED=OPTIONAL
